code/datums/uplink_item.dm

```diff
@@ -22,13 +23,35 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 				uplink_items[I.category] = list()
 
 			uplink_items[I.category] += I
+			if(I.limited_stock < 0 && !I.cant_discount && I.item && I.cost > 1)
```

Comment:
That's to stop the actual list of possible discounted items from including anything that already costs 1TC, since you can't discount that .

The max is used so that after discounting an item that gets picked out , it's new price can't be lower than 1TC.

Reply:
You're right, I misunderstood. Sorry.